### ISSUE TITLE
copy/paste tags?

### ISSUE THREAD

**Issue by brontosaurusrex** [jangler/taffy#4]
Can the tool be used to copy paste tags?

    tool in.flac out.mp3

**Comment by jangler** [comment]
No, it currently can't do that. I'm curious what your use case for this is, since I've never needed to do this, beyond maybe SoX mangling Unicode tags.

**Comment by brontosaurusrex** [comment]
The usual transcoding of FLAC to various lossy formats and I don't want to use ffmpeg.

**Comment by jangler** [comment]
~~Going to add this capability this weekend if I remember; there have been a couple times when I wanted to do this after the issue was opened. Sorry if it's far too late for you!~~

I don't know when I'll get around to this anymore.
